### PR TITLE
Feature paragraph mode

### PR DESCRIPTION
**Pull request by gurjit03**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

Split view paragraph mode
image

Non split view paragraph mode
image

New Icon for paragraph mode
image

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: src/js/components/Baani.js [hunk 1]
```diff
@@ -109,6 +124,7 @@ export default class Baani extends React.PureComponent {
 
   showShare = e => {
     const currentShare = document.querySelector('.showShare');
+    console.log(currentShare, " current share..");
```
Comment (plug-n-play) [inline src/js/components/Baani.js:127]: let's remove this

Comment (saintsoldierx) [reply src/js/components/Baani.js:127]: @gurjit03 agree with Amandeep Singh

**Comment by saintsoldierx** [comment]
If i go to SG and then click paragraph mode, then search for shabad, it still keeps paragraph mode. see example

image

File: src/js/components/Baani.js (hunk 1 above)
Comment (gurjit03) [reply src/js/components/Baani.js]: :+1: def

**Comment by saintsoldierx** [comment]
Make sure, if it autoscroll is on and scrolling, if i scroll manually up or down the page, it should not pause. It should keep going.
Seems like it's working on firefox but speed setting doesn't work or is very very slow.

**Comment by saintsoldierx** [comment]
Hukamnama, random shabad, and pages from Index also still show paragraph. They should be disabled.

**Pull request merged**